multiversx_sdk_cli/cli.py

```diff
@@ -166,6 +175,17 @@ def _handle_verbose_argument(args: list[str]):
         args.insert(0, verbose_arg)
 
 
+def _handle_log_level_argument(args: list[str]):
```

Comment:
Could have handled both `--verbose` and `--log-level` in a single function of the kind e.g. `_handle_global_arguments` or `_handle_universal_arguments`. Just an opinion.